cmd/fleet/cron.go

```diff
@@ -705,6 +705,7 @@ func newCleanupsAndAggregationSchedule(
 	ctx context.Context,
 	instanceID string,
 	ds fleet.Datastore,
+	lq fleet.LiveQueryStore,
```

Comment:
Looks like the `lq` argument (that was added for the redis cleanup job) is not needed anymore? We should remove it.